### ISSUE TITLE
Some of the functions do not appear to work...

### ISSUE THREAD

**Issue by jrbouldin** [meysubb/collegeballR#1]
For example:

team_stats(team_id=104, year=2019, sport="MBA", by = "game")
team_stats(team_id=104, year=2019, sport="MBA", by = "season")
team_stats(team_id="UC Santa Barbara", year="2019", sport="MBA", by = "season")

all return:
Error in if (sport == "MFB") { : argument is of length zero

Attempting to debug the problem, team_stats() returns "Error in nchar(sport) : argument "sport" is missing, with no default"

So basically, I can neither get the team stats, nor figure out why.  It doesn't work.

**Comment by meysubb** [comment]
Thanks for the issue. Think I know what the issue is. Working on making some changes!

**Comment by meysubb** [comment]
Couple of things: 

1) Current data doesn't let you pull 2019 data. Will make change
2) Underlying data comes from stat.ncaa and their structure has changed

**Comment by meysubb** [comment]
It should work now, give it a go. Try something like this

```R
team_stats(team_id=104, year=2019, sport="MBA", by = "season")
```
